### PR TITLE
Patternlab/DP-9693 modify body text color to use a darker grey

### PR DESCRIPTION
**Pull request by clair0917**
## Description
Switch body copy color to #141414 ($c-font-base: $c-black) while keeping requested elements #535353 ($c-font-detail: $c-theme-gray)

## Related Issue / Ticket

- [JIRA issue DP-9693](https://jira.mass.gov/browse/DP-9693)

## Steps to Test

1. Per https://jira.mass.gov/browse/DP-9693 open the document found here: https://docs.google.com/document/d/1DInGiHDJX945afVmS6m6czGD0oAJGJgBMI3p0vxjySc/edit

2. Check that body copy is now defined as  #141414 throughout, while elements details in the doc remain #535353

Backstop files are attached

### CODE REVIEW

File: assets/scss/01-atoms/_input-number.scss [hunk 1]
```diff
@@ -55,16 +55,16 @@
     transform: rotate(180deg);
   }
 }
-
-//theme
+
+//theme
```
Comment (Tregonia) [inline assets/scss/01-atoms/_input-number.scss:59]: Not a big deal, or something that is needed in this PR, but as I have went through the theme files, and this `//theme` section still exists, I have been trying to integrate them and clean up the files.

File: assets/scss/02-molecules/_callout-link.scss [hunk 2]
```diff
@@ -114,17 +114,17 @@
...
+    color: $c-font-base;
     font-weight: 700;
   }
 }
 
 .ma__callout-link--white{
   background-color: $c-white;
-  border-color: tint($c-theme-gray,70%);
+  border-color: tint($c-font-detail,70%);
   box-shadow: 0 0.25rem 0.5rem rgba(#000, 0.25);
```
Comment (Tregonia) [inline assets/scss/02-molecules/_callout-link.scss:129]: The black here could use the `$c-black` variable

File: assets/scss/02-molecules/_sticky-nav.scss [hunk 3]
```diff
@@ -147,8 +147,8 @@
     }
   }
 }
-
-//theme
+
+//theme
 .ma__sticky-nav {
   @media ($bp-medium-max) {
```
Comment (Tregonia) [inline assets/scss/02-molecules/_sticky-nav.scss:153]: I do not understand why the `@` here is colored red in this PR. Is there a change here?

File: assets/scss/03-organisms/_illustrated-header.scss [hunk 4]
```diff
@@ -120,40 +121,40 @@ $bp-illustrated-header-image-width: 360px;
...
+  }
+
+  &__category {
+    font-weight: 700;
+  }
+
+  .ma__page-header {
+
+    &__intro {
+    }
```
Comment (Tregonia) [inline assets/scss/03-organisms/_illustrated-header.scss:147]: Have an empty declaration here

File: assets/scss/02-molecules/_sticky-nav.scss (hunk 3 above)
Comment (clair0917) [reply assets/scss/02-molecules/_sticky-nav.scss:153]: I'm not sure why it does that either, actually.

**Review by Tregonia — APPROVED** [review]
Looks good

**Pull request merged**